docs/source/exercises.rst

```diff
@@ -214,9 +214,9 @@ We can verify this works by computing the *inverse* FFT on the output with NumPy
 
 .. testoutput:: qft
 
-    [ 0.00000000e+00+0.00000000e+00j  1.00000000e+00+5.45603965e-17j
+    [ 0.00000000e+00+0.00000000e+00j  1.00000000e+00+4.30636606e-17j
       0.00000000e+00+0.00000000e+00j  0.00000000e+00-1.53080850e-17j
-      0.00000000e+00+0.00000000e+00j -7.85046229e-17-2.39442265e-17j
+      0.00000000e+00+0.00000000e+00j -7.85046229e-17-1.24474906e-17j
       0.00000000e+00+0.00000000e+00j  0.00000000e+00-1.53080850e-17j]
```

Comment:
See [this note about precision](https://github.com/numpy/numpy/commit/22c56b5707a8980d216a95b50db9f880a18b1caa#diff-f8b7788a1ea69df4fa13d3a39f5139dc72a91b1e1c1df4476ae3c00de70d329bR4-R8):

> The various FFT routines in `numpy.fft` now do their calculations natively in
float, double, or long double precision, depending on the input precision,
instead of always calculating in double precision. Hence, the calculation will
now be less precise for single and more precise for long double precision.
> 
> The data type of the output array will now be adjusted accordingly.